### ISSUE TITLE
ISSUE: child locks not working

### ISSUE THREAD

**Issue by libots** [FutureTense/keymaster#157]
**Describe the bug**
Child lock isn't working. When creating the integration, the YAML file is recognized, but the integration fails to create the child lock or handle it at all in the generated package files. Additionally, when going back into options, the "path to child locks file" is empty.

**Environment (please complete the following information):**

- OS: HA docker container
- Type of system that HA is running on: NUC (Debian VM docker container)
- Home Assistant version: 2021.4.6
- Docker
- Component version: 0.0.31
- Z-Wave integration name: zwave_js
- Lock make and model: Yale YRD256 (master) and YRD226 (child)

**Logs**
No errors are generated on debug mode *unless* I try to specify a `door_sensor_entity_id` (I tried that after some googling, but it gives `ERROR (SyncWorker_35) [custom_components.keymaster.config_flow] Child locks file data is invalid: extra keys not allowed @ data['back_child']['door_sensor_entity_id']`).

**Additional context**
My child file:
```yaml
back_child:
  lock_entity_id: lock.back
  # door_sensor_entity_id: binary_sensor.door_back_homekit
  alarm_level_or_user_code_entity_id: sensor.fake
  alarm_type_or_access_control_entity_id: sensor.fake
```

**Comment by tykeal** [comment]
I've got a pair of Kwikset locks that are supposed to be in a parent / child relationship that I can volunteer for doing testing with.

I turned on debug logging for a bit and tried to see what was what, but there was no debug output other than all my locks being regularly refreshed.

I can further state that @libots is correct that the child lock file path is not kept across configuration runs. It _is_ however storing the information in the core.config_entries file and if I try to give a non-existent or non-conformant yaml file the config workflow recognizes it and throws up an error.
